test/qpy_compat/test_qpy.py

```diff
@@ -288,6 +288,58 @@ def generate_evolution_gate():
     return qc
 
 
+def generate_control_flow_circuits():
+    """Test qpy serialization with control flow instructions."""
+    from qiskit.circuit.controlflow import WhileLoopOp
+
+    # If instruction
+    circuits = []
+    qc = QuantumCircuit(2, 2)
+    qc.h(0)
+    qc.measure(0, 0)
+    with qc.if_test((qc.clbits[0], True)):
+        qc.x(1)
```

Comment:
builder

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7584/commits/261663865ff09c6243f9f3d6bbd22550189d7648